iac/terraform/aws/auth_cognito/lambdas.tf

```diff
@@ -0,0 +1,38 @@
+locals {
+  lambda_archive_path = "${path.module}/../../../../core/services/aws/.build/zips"
+  lambda_options = {
+    custom_email_template = {
```

Comment:
There should not be a lambda to only generate email template

Reply:
This lambda is not generating an Email template for bad naming it's creating confusion now I tried to give a better name
cognito_custom_message_trigger [Commit](https://github.com/CraftsmenLtd/BloodConnect/commit/28f045306270fdc86a03546bd3b3e08356c4dc74#diff-9fcfda62a75664be8204a906a8560f94308c1ba2b6628a3ee928aacda1a7d7f1)